pkgs/tools/networking/openfortivpn/default.nix

```diff
@@ -1,11 +1,11 @@
 { stdenv, lib, fetchFromGitHub, autoreconfHook, pkg-config
-, openssl, ppp
-, systemd ? null }:
+, openssl
+, ppp
+, systemd
+, withSystemd = stdenv.isLinux;
+, withPpp = stdenv.isLinux;
```

Comment:
```suggestion
, withSystemd ? stdenv.isLinux
, withPpp ? stdenv.isLinux
```
total derp of me

Reply:
honestly, i have no idea about nix (language)

just switched my macos over to nix (from my own set of bash scripts) to manage packages.

noticed this gem, tried with some tutorial changes, wanted to contribute.

thank you for your help and patience!